pkg/local_object_storage/metabase/control.go

```diff
@@ -106,7 +106,7 @@ func (db *DB) init(reset bool) error {
 		}
 
 		if !reset {
-			err = syncCounter(tx)
+			err = syncCounter(tx, db.epochState.CurrentEpoch(), false)
```

Comment:
Same here.

Reply:
Also, I don't see any changes in the `syncCounter` itself in this commit.

Reply:
fixed